src/objective-c/GRPCClient/GRPCCall.h

```diff
@@ -55,43 +57,48 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)didReceiveInitialMetadata:(nullable NSDictionary *)initialMetadata;
 
 /**
- * This method is deprecated and does not work with interceptors. To use GRPCCall2 interface with
- * interceptor, implement didReceiveData: instead. To implement an interceptor, please leave this
- * method unimplemented and implement didReceiveData: method instead. If this method and
- * didReceiveRawMessage are implemented at the same time, implementation of this method will be
- * ignored.
- *
  * Issued when a message is received from the server. The message is the raw data received from the
  * server, with decompression and without proto deserialization.
+ *
+ * <b> This method is deprecated and does not work with interceptors.</b> To use GRPCCall2
+ * interface with interceptor, implement didReceiveData: instead. To implement an interceptor,
+ * please leave this method unimplemented and implement GRPCResponseHander::didReceiveData method
```

Comment:
Nit: these 2 sentences seem to be saying the same thing:
```
To use GRPCCall2
 * interface with interceptor, implement didReceiveData: instead. To implement an interceptor,
 * please leave this method unimplemented and implement GRPCResponseHander::didReceiveData method
 * instead.
```
Consider rephrasing, maybe something like: To use GRPCCall2 interface with interceptor, implement didReceiveData: instead and  leave this method unimplemented.